### PR TITLE
Fix/Update brazilian_portuguese.xml v7.8

### PR DESCRIPTION
**Pull request by hdstec**
Fix of several sentences
Fix long sentences
General improvements (for easy understanding).

### CODE REVIEW

**Comment by chcg** [comment]
@hdstec Your PR contains an extra file update brazilian_portuguese.xml https://github.com/notepad-plus-plus/notepad-plus-plus/pull/6236/files#diff-3535869450ada0bd802fab9c1c2c8406. Is this intended? See also PR #6231.

**Comment by hdstec** [comment]
@chcg this extra file "update brazilian_portuguese.xml" was the first revision I made of the translation, but it is not complete.

For this reason, I have made a new revision for the translation (much more complete).
Please consider just this: [c32f24b](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/6236/commits/c32f24bb353be49609a4577c8662ed4c04224ea5 )

File: PowerEditor/installer/nativeLang/update brazilian_portuguese.xml [hunk 1]
```diff
@@ -0,0 +1,1252 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/update brazilian_portuguese.xml]: This file shouldn't be here.

**Comment by donho** [comment]
@fpaganini @altiereslima @xylographe @tabarra
@sergioraa @Andrulko @taveirajc Could anyone of you review and approve the modification please?

**Pull request closed without merging**